File: dados_h_e_g.py
# Obtenha dados da altura e o genero (H ou M) de 15 pessoas e apresente os seguintes resultados:
# - A maior e a menor altura do grupo;
# - A média de altura dos homens;
# - O numero de mulheres.

maior_h = 0
menor_h = 0
soma_altura_homens = 0
num_homens = 0
num_mulheres = 0

for i in range(4):
    altura = float(input(f"Informe a altura da pessoa {i+1}: "))
    genero = input(f"Informe o gênero da pessoa {i+1} (H ou M): ").upper()

    if altura > maior_h:
        maior_h = altura
    if altura < menor_h:
        menor_h = altura

    if genero == 'H':
        soma_altura_homens += altura
        num_homens += 1
    elif genero == 'M':
        num_mulheres += 1

media_altura_homens = soma_altura_homens / num_homens if num_homens > 0 else 0

print(f"A maior altura do grupo é: {maior_h}")
print(f"A menor altura do grupo é: {menor_h}")
print(f"A média de altura dos homens é: {media_altura_homens:.2f}")
print(f"O número de mulheres é: {num_mulheres}")

File: dados_altura_genero_js/dados_h_e_g.js
let maior_h = 0;
let menor_h = Infinity;
let soma_altura_homens = 0;
let num_mulheres = 0;

for (let i = 1; i <=15; i++){

    const altura = parseFloat(prompt(`Informe a altura da pessoa ${i}: `));
    const genero = prompt(`Informe o genero da pessoa ${i} (H ou M): `)

    if (altura > maior_h){
        maior_h = altura
    }
    
    if (altura < menor_h){
        menor_h = altura
    }

    if (genero.toUpperCase() === 'H'){
        soma_altura_homens +=altura;
    }
    else if (genero.toUpperCase() === 'M'){
    num_mulheres++;
    }
}

const media_h_homens = soma_altura_homens / (15 - num_mulheres);

console.log("A Maior alura do grupo é: ", (maior_h));
console.log("A menor altura do grupo é: ", (menor_h));
console.log("A media da altura dos homens é: ", (media_h_homens.toFixed(2)));
console.log("O numero de mulheres é: ", (num_mulheres));


